kvdb/go.mod

```diff
@@ -113,4 +137,6 @@ replace github.com/ulikunitz/xz => github.com/ulikunitz/xz v0.5.11
 // https://deps.dev/advisory/OSV/GO-2021-0053?from=%2Fgo%2Fgithub.com%252Fgogo%252Fprotobuf%2Fv1.3.1
 replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 
-go 1.19
+replace github.com/lightningnetwork/lnd/sqldb => ../sqldb
+
+go 1.21.4
```

Comment:
I think we want to keep this at Go 1.19 as we hadn't yet bumped the Go versions repo wide yet.

Reply:
This is bumped because `kvdb` now depends on `sqldb` which is 1.21.4. On the other hand I think we're safe to bump as the main go.mod also is using 1.21.4.